server/sonar-web/src/main/js/apps/create/components/OrganizationKeyInput.tsx

```diff
@@ -113,7 +113,7 @@ export default class OrganizationKeyInput extends React.PureComponent<Props, Sta
         label={translate('onboarding.create_organization.organization_name')}
         required={true}>
         <div className="display-inline-flex-baseline">
-          <span className="little-spacer-right">
+          <span className="little-spacer-right min-width-fit-content">
```

Comment:
CD-2226 To avoid text-wrap for EU instance url 'https://app-eu.codescan.io/' on Org Creation page

Reply:
Replace with `little-spacer-right nowrap`